src/utils/helmetPolyfill.ts

```
@@ -63,39 +63,69 @@ function helmet(options: HelmetOptions = {}) {
+      "script-src-attr 'none'",
+      "style-src-attr 'none'",
+      // SECURITY: Require trusted types for DOM manipulation
+      "require-trusted-types-for 'script'",
+      "trusted-types default"
     ].join('; ');
 
-    // SECURITY: Comprehensive security headers
+    // SECURITY: Comprehensive security headers with enhanced protection
     const securityHeaders: Record<string, string> = {
       'Content-Security-Policy': cspDirectives,
       'X-Content-Type-Options': 'nosniff',
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)